apps/full-site-editing/full-site-editing-plugin/newspack-blocks/blog-posts-block-editor.js

```diff
@@ -31,7 +32,7 @@ addFilter(
 registerBlockType( blockName, {
 	...settings,
 	title: __( 'Blog Posts', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **80** times:
`translate( 'Blog Posts' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/fse-blocks-categories-collections)